contents/handbook/company/team/paolo-damico.md

```diff
@@ -4,8 +4,26 @@ sidebar: Handbook
 showTitle: true
 ---
 
+## Q1 2022 Goals
+- **Product strategy.**
+  - 2x ambitious Product milestones (like Experimentation). Strategy >> Execution. Exciting/evangelism.
+  - Decentralize & scale customer feedback loop with Customer Success Team.
+  - Scale (make sure it works optimally) Design work with Eng Teams.
+  - Bring more industry context. TODO:
+  - Great handle on our numbers. Discover opportunities in data (increase Discoveries from this). Excite others to do this.
```

Comment:
Yep I think a great specific (but perhaps tough) goal here, would be to have an answer for why one (or more) of the company metrics have gone up or down each week during the all hands.

Reply:
+1 - this _could_ be really fascinating. Maybe we lack the data volume, but we'll quickly figure that out if so.

Reply:
Had a chat with Marcus today and will focus on this but with a different mindset: working out a single ambiguous opportunity to improve Discoveries.